frontend/public/components/edit-yaml.jsx

```diff
@@ -50,13 +50,20 @@ const stateToProps = ({ k8s, UI }) => ({
   models: k8s.getIn(['RESOURCES', 'models']),
 });
 
+const WithYamlTemplates = (Component) =>
+  function Comp(props) {
+    const [templateExtensions] = useResolvedExtensions(isYAMLTemplate);
```

Comment:
Ideally we don't resolve all templates when we only actually need a specific one.
Again, you can augment the filter function.